sources/packages/api/src/services/disbursement-schedule-service/disbursement-schedule-service.ts

```diff
@@ -329,8 +342,34 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
         { coeThresholdDate },
       );
     }
-    coeQuery.orderBy("disbursementSchedule.coeStatus");
-    return coeQuery.getMany();
+    if (paginationOptions.searchCriteria) {
+      coeQuery.andWhere(
+        new Brackets((qb) => {
+          qb.where(
+            "CONCAT(user.firstName,  ' ', user.lastName ) Ilike :searchCriteria",
```

Comment:
What exactly the search by name is supposed to work here?
Can it not search by firstName or lastName?

Reply:
This will work for search by firstname + search by lastName + search by fullName.

Re-used the same logic from user data table search.

Reply:
Extra space removed as discussed.